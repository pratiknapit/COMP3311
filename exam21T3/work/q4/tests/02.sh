psql property -c 'select address(46444);'
