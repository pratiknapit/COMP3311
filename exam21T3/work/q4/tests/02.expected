                address                
---------------------------------------
 76 Spring Street, Bondi Junction 2022
(1 row)
